src/code42cli/cmds/alert_rules.py

```diff
@@ -31,11 +34,14 @@ class AlertRuleTypes:
 _HEADER_KEYS_MAP["ruleSource"] = "Source"
 _HEADER_KEYS_MAP["isEnabled"] = "Enabled"
 
+DEPRECATION_TEXT = "Incydr functionality is deprecated. Use the Incydr CLI"
```

Comment:
Looks like this is missing the end of the sentence that the others have?

Reply:
Fixed!